<ui version="4.0" >
 <class>EditCameraClass</class>
 <widget class="QDialog" name="EditCameraClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>245</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>EditCamera</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="Camera" >
     <property name="title" >
      <string>Camera List</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QListWidget" name="listWidget" >
        <property name="maximumSize" >
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Current Camera Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineEdit" />
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineEdit_2" />
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="lineEdit_4" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Up Vector</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="Camera_2" >
        <property name="text" >
         <string>Target </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lineEdit_3" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="margin" >
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Add" >
       <property name="focusPolicy" >
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="focusPolicy" >
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3" >
       <property name="focusPolicy" >
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Close" >
       <property name="focusPolicy" >
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
